.github/workflows/integration.yaml

```diff
@@ -207,6 +192,54 @@ jobs:
             unicode \
             unicode/utf16 \
             unicode/utf8
+          do
+            tinygo test -target wasi -c -o ${{ env.BINARY_OUT }}/${value/\//_}.test $value
+          done
+
+      - name: Checkout wazero
+        if: steps.check-exist.outputs.exist == 'false'
+        uses: actions/checkout@v3
+
+      - name: Upload tinygo-test-binary
+        if: steps.check-exist.outputs.exist == 'false'
+        uses: actions/upload-artifact@v3
+        with:
+          name: tinygo-test-binaries-${{ env.TINYGO_VERSION }}
+          path: |
+            ${{ env.BINARY_OUT }}/*.test
+
+  tinygo:
+    needs: build_tinygo_test_binary
+    name: TinyGo (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false # don't fail fast as sometimes failures are arch/OS specific
+      matrix:
+        os: [ubuntu-20.04, macos-12, windows-2022]
+
+    steps:
+      - name: Checkout wazero
+        uses: actions/checkout@v3
+
+      - name: Download the built binary
+        run: gh run download --name tinygo-test-binaries-${{ env.TINYGO_VERSION }}
+        env:
+          # Necessary to use gh command.
+          GH_TOKEN: ${{ github.token }}
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Install wazero
+        run: go install ./cmd/wazero
+
+      - name: Run standard library tests
+        run: |
+          for bin in *.test; do
```

Comment:
it turns out that the majority of the time was spent on compiling wasm binary by tinygo, so it seems all test finishes in like 30 sec or so.

Reply:
good news cc @dgryski